### PR TITLE
[READY-TO-REVIEW] Document publishing conventions

### PR DESCRIPTION
**Pull request by jarusso**
Document our plans for using github and for categorization

fixes issue 31299
and issue 31300

### CODE REVIEW

**Comment by jarusso** [comment]
@miguelestrada please review

**Comment by jarusso** [comment]
@miguelestrada bump

File: V1_EngagingWithUs.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+for developers to use, it will be removed and calls to this API will no longer work. They will be kept on to provide backward 
+compatibility, and to provide time for developers to update affected code.
+
+You will see these various categories called out on those specific APIs. All other, undecorated API references are 
+considered published and are fully supported. 
+
+To provide more transparency in how we’re working and what we’re building, we will surface our content in markdown files in 
+the github repo - [github.com/watsonwork/watsonwork-developer-docs](https://github.com/watsonwork/watsonwork-developer-docs) so that you can engage with us through github to inform 
+and provide feedback to help make and design great APIs.
+
```
Comment (miguelestrada) [inline V1_EngagingWithUs.md:27]: Include here a paragraph on how and where to provide feedback?

- something wrong / incorrect / confusing with doc and/or with the results experienced with an API
- a reaction or comment to a experimental, beta, future API proposal / doc
- a suggestion for something to be included / modified
- other ?

File: V1_EngagingWithUs.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Engaging with us
+
+We are continually adding value and want to include you as early as possible in our development process so we can include 
+your input and feedback. Some features are stable (in that they will not change), others are in progress of being built which we call **beta** 
+and others are things we want to try and discover what would work best, we call **experimental**. Additionally, when we are about to 
+introduce something new and want to get feedback on the concepts and understanding before we expose API’s, so that your 
+feedback can inform and guide the future API, we will call these **future**. Also, **future** may, (but not always), include the ability for you to 
+play with an API, through one of our tools (like the GraphQL Explorer), so you can have some experience with it and give us 
+feedback to help inform and improve things.
+
```
Comment (miguelestrada) [inline V1_EngagingWithUs.md:16]: Add here a reference to how to change the level of APIs exposed in API Explorer.  We just delivered this functionality which should also be referenced in the "What's New" doc.  @nivekselig can provide details

File: V1_EngagingWithUs.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+keep in the github repository mentioned above. When we build, we only bring in those files that are marked as **published**, **beta** 
+or **deprecated**, and do not add those marked as **future** or **experimental**.
+
+**Folder Structure**
+
+Root: 
+- [Navigation Menu](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/NavigationMenu.md)
+- [Get Started](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/V1_GetStarted.md)
+- [What's New](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/Whats_New.md)
+
```
Comment (miguelestrada) [inline V1_EngagingWithUs.md]: We have other entries in the root.... was the above a sample or is it supposed to be complete.

File: V1_EngagingWithUs.md (hunk 1 above)
Comment (miguelestrada) [reply V1_EngagingWithUs.md:27]: also include that in addition to above feedback we welcome Pull Requests with suggested changes, tutorials, how-tos, example references, etc.

**Review by miguelestrada — CHANGES_REQUESTED** [review]
see comments

File: V1_EngagingWithUs.md (hunk 3 above)
Comment (jarusso) [reply V1_EngagingWithUs.md]: I don't know what you mean here

File: V1_EngagingWithUs.md (hunk 2 above)
Comment (jarusso) [reply V1_EngagingWithUs.md:16]: ok, all set

File: V1_EngagingWithUs.md (hunk 1 above)
Comment (jarusso) [reply V1_EngagingWithUs.md:27]: ok all set

File: V1_EngagingWithUs.md (hunk 3 above)
Comment (miguelestrada) [reply V1_EngagingWithUs.md]: You are providing info on folder structure and call out 3 items at the "Root:" level (Navigation Menu, Get Started, What's New).  We have more than 3 items at the root level already.

**Comment by jarusso** [comment]
@miguelestrada can we merge this, it's ready

**Review by miguelestrada — APPROVED** [review]

**Pull request merged**